internal/storage/servers.json

```diff
@@ -117978,6 +117978,1492 @@
+      },
+      {
+        "country": "gw2",
+        "region": "tpe1",
+        "city": "slickvpn",
+        "hostname": "gw2.tpe1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "103.98.73.10"
+        ]
+      },
+      {
+        "country": "gw2",
+        "region": "yyz1",
+        "city": "slickvpn",
+        "hostname": "gw2.yyz1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "172.105.9.120"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "dfw3",
+        "city": "slickvpn",
+        "hostname": "gw3.dfw3.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "104.237.128.245"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "ewr1",
+        "city": "slickvpn",
+        "hostname": "gw3.ewr1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "50.116.54.120"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "iad1",
+        "city": "slickvpn",
+        "hostname": "gw3.iad1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "185.151.12.242"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "lax3",
+        "city": "slickvpn",
+        "hostname": "gw3.lax3.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "107.160.101.199"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "lhr1",
+        "city": "slickvpn",
+        "hostname": "gw3.lhr1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "37.123.117.149"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "lhr2",
+        "city": "slickvpn",
+        "hostname": "gw3.lhr2.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "139.162.220.234"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "pao1",
+        "city": "slickvpn",
+        "hostname": "gw3.pao1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "173.255.243.165"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "per1",
+        "city": "slickvpn",
+        "hostname": "gw3.per1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "103.231.89.206"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "sou1",
+        "city": "slickvpn",
+        "hostname": "gw3.sou1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "78.143.255.175"
+        ]
+      },
+      {
+        "country": "gw3",
+        "region": "yyz1",
+        "city": "slickvpn",
+        "hostname": "gw3.yyz1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "198.144.157.100"
+        ]
+      },
+      {
+        "country": "gw4",
+        "region": "lhr1",
+        "city": "slickvpn",
+        "hostname": "gw4.lhr1.slickvpn.com",
+        "udp": true,
+        "ips": [
+          "146.185.17.165"
+        ]
+      },
+      {
+        "country": "gw4",
```

Comment:
Please also set the server `VPN` field to `vpn.OpenVPN`, this should show in the JSON data.
(this is a recent change from last week)

Reply:
Added `VPN` field to to the `servers.go` output.